packages/replay-next/pages/variables.css

```diff
@@ -48,7 +48,7 @@
   --primary-accent: #01acfd;
   --primary-accent-hover: #0194ff;
 
-  --secondary-accent: var(--secondary-accent);
+  --secondary-accent: #d72451;
```

Comment:
Ignore my initial, stupid comment.

Anyway, after this PR both `--secondary-accent` and `--secondary-accent-hover` are the same color, which seems wrong.

Reply:
Will investigate